common/usr/lib/systemd/system/postfix-create-DH2048.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Create the DH2048 group for postfix
-ConditionPathExists=!/etc/pki/dovecot/private/postfix.dh.param
+ConditionPathExists=!/etc/postfix/postfix.dh.param
 
```

Comment:
If we change the path we need to generate a new group.

What about moving the existing one (if we have it) to the new location?

Reply:
DH is ephemeral, at least we could remove it....but does it worth it

Reply:
We clean the old DH group

Reply:
hello @DavidePrincipi, could you check now the old group is removed if find